packages/webview_flutter/lib/src/platform_view_tizen.dart

```diff
@@ -591,6 +591,7 @@ class _TizenPlatformTextureView extends LeafRenderObjectWidget {
     required this.controller,
     required this.hitTestBehavior,
     required this.gestureRecognizers,
+    // ignore: unused_element
     this.clipBehavior = Clip.hardEdge,
```

Comment:
https://github.com/flutter/flutter/commit/e0ac5da1647aed8e4113d80d11b663fa161ad5cd#diff-7cd9c238c904111d0fa1ae7a20a44c8bf24e802eb1f881217b60f84cd76e2b92

Adding missing lines to `_TizenWebViewState` and `TizenView` seems to be a right fix.

Refer to [this comment](https://github.com/flutter-tizen/plugins/pull/54#pullrequestreview-625891709) for comparison between `AndroidWebView` and `TizenWebView` and related types.